AutoDockTools/MoleculePreparation.py

```diff
@@ -209,7 +209,7 @@ def addCharges(self, mol, charges_to_add):
         #MODE SWITCH 2: adding charges????  ||NOT IN USE||
         if debug: print("self.chargeType=", self.chargeType)
         if self.chargeType is None:
-            len_zero_charges = len([x for x in mol.chains.residues.atoms if x.charge==0])
+            len_zero_charges = len([x for x in mol.chains.residues.atoms if x._charges.values()=={}])
```

Comment:
Can you elaborate here please?

Reply:
Usually ```self.chargeType is None``` condition is ever met. 
However, I had a problem, where the molecule hasn't charged, so x.charge ere None and ```x.charge==0``` fail. 
So, I find this hack accessing the x._charges dictionary. 

![image](https://user-images.githubusercontent.com/32571404/129788080-4d89018e-aa75-4927-b399-cf0fafb20338.png)

However, I've debugged the prepare functions with different parameters and I haven't met ```self.chargeType is None```.  So I recommend discard this change, and keep an eyes if someone ever open an issue.